src/pages/Dashboard/scenes/Emails/Emails.tsx

```diff
@@ -64,19 +68,65 @@ const Emails: React.FC = () => {
   };
 
   const handleSelectChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    let selectedMentors: Mentor[] | undefined = [];
+    let selectedMentees: Mentee[] | undefined = [];
     switch (e.target.value) {
       case 'allMentors':
-        break;
-      case 'allMentees':
+        selectedMentors = mentors;
         break;
       case 'acceptedMentors':
+        selectedMentors = mentors?.filter(
+          (mentor) => mentor.state === 'approved'
```

Comment:
We have an enum for this ApplicationStatus. Let's use that so it's easier for us to update it the naming got changed 

```suggestion
          (mentor) => mentor.state === ApplicationStatus.APPROVED
```